lib/utils/password_form_field.dart

```diff
@@ -0,0 +1,42 @@
+import 'package:flutter/material.dart';
+
+class PasswordFormField extends StatelessWidget {
+  final TextEditingController? controller;
+  final bool? obscureText;
```

Comment:
You can make it non-nullable.
```suggestion
  final bool obscureText;
```